### PR TITLE
[Entry] Java Hashmap: Create the term entry map.containsKey()

### LINKED ISSUE #4180: [Entry] Java Hashmap: Create the term entry `map.containsKey()`
### Reason/inspiration (optional)

We want to add the `.containsKey()` method as a new term entry for [HashMap](https://www.codecademy.com/resources/docs/java/hashmap). Provide the entry in a markdown file and place it under the file path `content/java/concepts/array-list/terms/containsKey/containsKey.md`

The entry should have the following:
– An introductory paragraph about what the method does.
– A syntax section with a pseudo code block explaining it.
– An example showing the method in use.

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by andersooi**
### Description

- Created new term entry for Java `map.containsKey()`
- Closes #4180 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/map/terms/containsKey/containsKey.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+Title: '.containsKey()'
+Description: 'Checks if a map contains a specified key'
```
Comment (dakshdeepHERE) [inline content/java/concepts/map/terms/containsKey/containsKey.md]:
```suggestion
Description: 'Checks if a map contains a specified key.'
```

File: content/java/concepts/map/terms/containsKey/containsKey.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.containsKey()` method is declared in the [`Map`](https://www.codecademy.com/resources/docs/java/map) interface and is implemented in the [`HashMap`](https://www.codecademy.com/resources/docs/java/hashmap) class. It is used to determine if a `Map` object contains a specific key. The function returns a boolean value: `true` if the key exists, and `false` if not.
+
+## Syntax
+
+The `.containsKey()` method can be called on a `HashMap` instance and it requires one parameter, the key that needs to be checked.
```
Comment (dakshdeepHERE) [inline content/java/concepts/map/terms/containsKey/containsKey.md:19]:
```suggestion
The `.containsKey()` method can be called on a `HashMap` instance and it requires one parameter, the key that needs to be checked:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few minor changes

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

File: content/java/concepts/map/terms/containsKey/containsKey.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Hash Maps'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.containsKey()` method is declared in the [`Map`](https://www.codecademy.com/resources/docs/java/map) interface and is implemented in the [`HashMap`](https://www.codecademy.com/resources/docs/java/hashmap) class. It is used to determine if a `Map` object contains a specific key. The function returns a boolean value: `true` if the key exists, and `false` if not.
```
Comment (avdhoottt) [inline content/java/concepts/map/terms/containsKey/containsKey.md:15]:
```suggestion
The `.containsKey()` method is declared in the [`Map`](https://www.codecademy.com/resources/docs/java/map) interface and is implemented in the [`HashMap`](https://www.codecademy.com/resources/docs/java/hashmap) class. It is used to determine if a `Map` object contains a specific key. The function returns a boolean value `true` if the key exists, and `false` if not.
```

**Review by avdhoottt — APPROVED** [review]
Excellent entry 🚀 Looks good to get merged!!

**Pull request merged**